www/docs/src/content/docs/0.1.0-beta.5/start-here/getting-started.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Getting Started with Astro StudioCMS
+description: Get started building with Astro StudioCMS.
+banner:
+  content: |
+    We've renamed the package to <b>studiocms</b> 🎉
+slug: 0.1.0-beta.5/start-here/getting-started
+---
+
+import { PackageManagers } from 'starlight-package-managers';
+
+## Let's get started
+
+To start using Astro StudioCMS, you'll need:
+
+* An Astro project
+* An Astro Studio account
+* An Astro Studio project
+* The Astro StudioCMS integration
+
+## Creating an Astro Studio account
+
+:::note
```

Comment:
I'd probably make this an error or red or as big and fuckin eye-catching as possible. A 📣 emoji wouldn't go amiss too!

Reply:
This is the old version of the docs.... (beta.5)

Reply:
changed to danger on latest docs... [eb86c4b](https://github.com/astrolicious/studiocms/pull/281/commits/eb86c4bd46eb89833f5ef5aac8098db8e180e5e6)